aws/resource_aws_wafv2_web_acl.go

```diff
@@ -253,7 +252,7 @@ func resourceAwsWafv2WebACLUpdate(d *schema.ResourceData, meta interface{}) erro
 			Scope:            aws.String(d.Get("scope").(string)),
 			LockToken:        aws.String(d.Get("lock_token").(string)),
 			DefaultAction:    expandWafv2DefaultAction(d.Get("default_action").([]interface{})),
-			Rules:            expandWafv2Rules(d.Get("rule").(*schema.Set).List()),
+			Rules:            expandWafv2WebACLRules(d.Get("rule").(*schema.Set).List()),
```

Comment:
addresses `TestAccAwsWafv2WebACL_ManagedRuleGroupStatement` and `TestAccAwsWafv2WebACL_ManagedRuleGroupStatemen` test failures as now they actually attempt to do an `update` vs. to `destroy/create` and it exposes the bug reported in #14035